torch/csrc/autograd/python_variable.cpp

```diff
@@ -462,37 +462,6 @@ static int THPVariable_clear(THPVariable* self) {
     //        because Tensor asked us to (it's already destructing).
 
     if (!self->cdata.unsafeIsBorrowed()) {
-      // TODO: empirically, on OS X this assert appears to be untrue
```

Comment:
Why did you remove this comment?

Reply:
I removed this for this PR and reenabled the assert in a separate PR (#78658)